### ISSUE TITLE
Add tests and notebook examples for new IODP functionality

### ISSUE THREAD

**Issue by moonshoes87** [PmagPy/PmagPy#580]
I'd like to get updated tests and notebook examples for the IODP functionality that has been updated.  

A few old things are now broken or changed -- for example, convert_2_magic.iodp_dscr_magic no longer exists, so Pmag GUI and iodp_dscr_magic.py need to be updated with the current stuff.

To do:
- Add or update tests for the new convert functions using the up-to-date data files in pmagpy_tests.test_imports3
- Add examples for the new convert functions into PmagPy_MagIC.ipynb (or put links into the Cookbook and PmagPy_introduction.ipynb that go to PmagPy_iodp_HOLE_template.ipynb)
- Put up-to-date IODP functions into Pmag GUI
- Remove out-of-date IODP functions from convert_2_magic

@ltauxe, do you want me to work on any of these things?  I might need a little bit of information from you, i.e. if the new convert_2_magic iodp functions are one-to-one replacements for the old ones, and which if any old functions to keep around.

**Comment by ltauxe** [comment]
just added the iodp_magic folder in data_files which has  a complete (almost) set of practice data (not affected by embargo) as examples.  what else do you need.  there is also an example notebook for that data set as well as a word document explaining how to use it out on the ship.  

the rest of this makes sense and I can slowly work on it over the next few weeks.  
lisa

**Comment by moonshoes87** [comment]
Thanks @ltauxe.  I'll look at your example notebook and see if I can use it to add tests that run on Travis CI.

**Comment by moonshoes87** [comment]
I added simple tests for each of the 5 conversions, just to make sure they run without errors.  

iodp_dscr_lore should replace iodp_dscr_magic
iodp_srm_lore should replace iodp_srm_magic
iodp_jr6_lore should replace iodp_jr6
iodp_samples_csv should replace iodp_samples
iodp_kly4s_lore doesn't replace anything

@ltauxe, unless you tell me otherwise, I will:

- add the new conversion functions into Pmag GUI 
- replace the old functions with new in the the command line programs (iodp_dscr_magic, iodp_srm_magic, iodp_jr6_magic, and iodp_samples_magic)
- replace old examples with new examples in PmagPy_MagIC.ipynb (alternatively I can remove old examples and just link users to the IODP template notebook...)
- get rid of the old functions

**Comment by ltauxe** [comment]
i agree with all of these.  thanks so much!  did you find the sample data files?  they are in data_files/iodp_magic
lisa

**Comment by moonshoes87** [comment]
yes, I found them!

**Comment by moonshoes87** [comment]
Remaining to do:

- add iodp_jr6_lore to Pmag GUI
- add iodp_kly4s_lore to Pmag GUI
- make command line program iodp_kly4s_magic
- decide about deprecating old functions (MagIC 2 issue)

**Comment by moonshoes87** [comment]
@ltauxe, I've put all of the IODP scripts into Pmag GUI.  Could you try them out and make sure they seem to be implemented as expected?
